STRUCTURING_YOUR_PROJECT.md

```diff
@@ -1,5 +1,161 @@
 # Structuring Your Project
 
+This page outlines good practices for Python based projects, but many of the principles discussed here can be applied to 
+other software projects as well.
+
+## Start from a common skeleton which separates key package components
+
+Packages should rely on a common file structure which separates the various elements of a package into separate 
+locations. Remember that a package is not just the code needed to carry out science, but also documentation and 
+deployment. We recommend the following skeletal file structure for some package `examplepackage`:
+ 
+```
+devtools/
+  README.md
+  travis-ci/
+    install_miniconda.sh
+  appveyor-ci/
+    run_with_env.cmd
+  conda-recipe/
+    meta.yaml
+    build.sh
+    bld.bat
+    README.md
+docs/
+examplepackage/
+  tests/
```

Comment:
Might be good to highlight where to put test data as well (`examplepackage/tests/data/` or `examplepackage/data/`?)